scene/gui/container.cpp

```diff
@@ -95,6 +95,7 @@ void Container::_sort_children() {
 
 void Container::fit_child_in_rect(Control *p_child, const Rect2 &p_rect) {
 
+	ERR_FAIL_COND(p_child == null);
```

Comment:
`null` in GDScript... :)

Reply:
You can use `!p_child`, which would likely be the most idiomatic in Godot's code base.

Reply:
Fixed